src/main/java/com/almanza/inventorystore/controller/ProductController.java

```diff
@@ -16,14 +16,14 @@
 public interface ProductController {
```

Comment:
Add Unit Tests for everything https://howtodoinjava.com/spring-boot2/testing/rest-controller-unit-test-example/